### ISSUE TITLE
Bump for WebTransport

### ISSUE THREAD

**Issue by TroyKomodo** [hyperium/h3#148]
https://web.dev/webtransport
https://w3c.github.io/webtransport/
https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3

Related to #71 and #129 
Chrome 97 has released (we are currently on 107)
Perhaps its time to reconsider the implementation for WebTransport.
I do not mind helping with the implementation.

**Comment by D1plo1d** [comment]
@TroyKomodo I am also interested in getting WebTransport support in h3. I got a bit ahead of myself and tried a naive approach of just adding the WebTransport-relevant settings the other day:

https://github.com/hyperium/h3/compare/master...D1plo1d:h3:webtransport-best-attempt#diff-3c80710a05f2e4cedfab749962e12c594ed9b1ec8b88d2c8712d4b22d2e81060R118 

I wasn't able to connect to the H3 WebTransport from Chrome but I did see some strange Settings frames coming in before the connections timed out (it looks like the second half of the settings are missing when compared with what Quiche receives from Chrome / WebTransport? Any idea why this would happen?).

What are the high level things that would need to get done in order to bring WebTransport support to h3?

So far I imagine we'll need to:
- Add settings (`ENABLE_WEB_TRANSPORT` and `SETTINGS_ENABLE_CONNECT_PROTOCOL`)
- On `CONNECT` requests establish a new WebTransport session
- Add the WebTransport frame types
